### PR TITLE
Use custom signin for most of E2E tests

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Handle page reload auto signin

Add user2 and use it for custom auth

[stage-2]

## Motivation and Context
Attempt to improve e2e test performance. Reduce the amount of tests that run with the Google account.

## How Has This Been Tested?
Ran e2e tests and verified various scenarios for failures.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
I did not update the hotmail account. I can do it for the next PR or just drop that; let me know your thoughts.

@ezequielc please review. Thanks!

File: test/e2e/common-header/cases/user-settings.js [hunk 1]
```diff
@@ -46,7 +46,7 @@
       it("should show user information", function() {
         expect(userSettingsModalPage.getFirstNameField().getAttribute('value')).to.eventually.equal("Jenkins");
         expect(userSettingsModalPage.getLastNameField().getAttribute('value')).to.eventually.equal("Rise");
-        expect(userSettingsModalPage.getEmailField().getAttribute('value')).to.eventually.equal("jenkins@risevision.com");
+        expect(userSettingsModalPage.getEmailField().getAttribute('value')).to.eventually.equal("jenkins.rise+custom@gmail.com");
```
Comment (ezequielc) [inline test/e2e/common-header/cases/user-settings.js]: @alex-deaconu Should we replace this with the env variable? Thanks!

Comment (alex-deaconu) [reply test/e2e/common-header/cases/user-settings.js]: @ezequielc I was thinking of that. Alternatively we can set a dummy email for that account (this is the email field on the account field). Let me know which you'd prefer. Thanks.

Comment (ezequielc) [reply test/e2e/common-header/cases/user-settings.js]: @alex-deaconu I would prefer to use the variable. Setting a dummy address could cause confusion. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu By the way, I am good with dropping the idea of a variable for the hotmail account. Thanks!

**Pull request merged**